docs/contribute-to-romcal.md

```diff
@@ -125,3 +125,51 @@ _pass the `-t` flag and the path of the file_
 ## Coverage reporting
 
 A coverage report is generated for every core branch build and published to [gh-pages]().
+
+## Committing changes
+
+### Message Format
+
+When you are ready to commit your changes, your commit message should follow the conventional commit format
+
+General format/terminology:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+Here are some guardrails to follow:
+
+When "type" one of the keys, the scopes allowed are as follows:
+
+- `ci` (updates to automation or the release process): No scope allowed
+- `build` (updates to build scripts): No scope allowed
+- `docs` (documentation): No Scope allowed
+- `feat` (net-new feature):
+  - Any of the calendar ids (`general` or `europe`, or `slovakia`, etc.)
+  - Any of the locale ids (`en`, `fr`, `es`, etc.)
+  - 'l10n' for changes to localization, either multiple locales or not specific to one
+  - 'calendar' for changes to calendars, either multiple or not specific to one
+- `enh` (enhancement): see `feat`
+- `fix` (bug fix): see `feat`
+- `refactor` (refactoring of code): see `feat`
+- `perf` (performance improvements): see `feat`
+- `test` (adding or updating tests): see `feat`
+- `style` (formatting, missing semi-colons, etc; no code change): see for `feat`
+
+### Automatic linting
+
+(of staged files)
+
+Files you stage & commit are automatically linted, and fixes applied.
+
+If it fails linting, your commit will fail.
+
+### Dependency Deduping
+
+Will be performed automatically, but you are responsible for committing it at your convenience.
```

Comment:
Missing subject in the first clause.

```suggestion
Dependency deduping will be performed automatically, but you are responsible for committing it at your convenience.
```

Reply:
it's literally already the title, but ok.... this is a common shorthand.

Reply:
Agreed that it is a common shorthand, however, I am for using full sentences.